tasks/engine_setup.yml

```diff
@@ -25,7 +25,7 @@
 
   - name: Update setup packages
     package:
-      name: "ovirt*setup*"
+      name: "ovirt*setup"
```

Comment:
I think that recommended way is to update 'ovirt*setup*' packages. Didi/Sandro?

@sandrobonazzola @didib

Reply:
the right way is to do only 'yum update ovirt-engine-setup' as far as I know. I can change it if needed.

Reply:
I think that have changed in 4.x, see step 3 at [1]. But let's confirmation from Sandro/Didi on that.

[1] https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.1/html/upgrade_guide/chap-updates_between_minor_releases#Upgrading_between_Minor_Releases

Reply:
ok, so we blocked on yum issue for now https://github.com/ansible/ansible/issues/34867

Reply:
This is still not working, we get dependency issues.
For example Error: Package: cockpit-storaged-157-1.el7.noarch (extras)
           Requires: storaged-iscsi >= 2.1.1
           Available: storaged-iscsi-2.5.2-2.el7.x86_64 (rhv-4.1.9)
               storaged-iscsi = 2.5.2-2.el7
           Available: storaged-iscsi-2.5.2-4.el7.x86_64 (rhv-4.2.1)
               storaged-iscsi = 2.5.2-4.el7

when running yum install.
we can split install and upgrade and use tags.
Any ideas?

Reply:
Does upgrade work if you execute it from command line? Because that seems to me like repository issue not related to this role

Reply:
Correct me if I am wrong, but for some reason the `yum install ovirt*setup*` command that the yum module is triggering in the case is not properly resolving the dependencies, and result failing on missing packages that actually exist.

I believe we verified locally that `yum update  ovirt*setup*`works without any problem.

If anyone with more knowledge on yum pkg manager could explain this behavior it would be good, but I think for now, it would be enough to change the yum module with shell module, and run the `yum update` instead.

Reply:
@KKoukiou you are right, yum update works work the original value, yum install not.
But if we want to use yum update we need to split it, AFAIK the one used for engine setup and upgrade.

Reply:
@myakove I think the package ovirt-engine-setup will be always there, since install tasks for ovirt-engine package is run first in every case, and brings the setup package as dependency. 

https://github.com/oVirt/ovirt-ansible-engine-setup/blob/b7201ec171df1b778e27d398f8bd5e369e50caf7/tasks/install_packages.yml#L1-L6

Reply:
Correction: 'Update setup packages' called only when ovirt_engine_setup_update_setup_packages is True so we can use shell and call yum update.
Should I do that?

Reply:
@myakove I am ok with this change.